### PR TITLE
[3120] Build "service updates" homepage summary and section

### PR DESCRIPTION
**Pull request by mooktakim**
### Context

https://trello.com/c/yMLVAa29/3120-l-build-service-updates-homepage-summary-and-section

### Changes proposed in this pull request

* New component added
* New section `/service-updates`
* YAML file `db/service_updates.yml` stores all the service updates
* 

### Guidance to review

* Start page, see most recent update
* Click see all updates

### CODE REVIEW

File: app/views/pages/start.html.erb [hunk 1]
```diff
@@ -8,12 +8,8 @@
...
   <div class="govuk-grid-column-two-thirds-from-desktop">
-    <p class="govuk-body"><%= t(".bullet_desc") %></p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li><%= t(".bullets.1") %></li>
-      <li><%= t(".bullets.2") %></li>
-      <li><%= t(".bullets.3") %></li>
-      <li><%= t(".bullets.4") %></li>
-    </ul>
+    <h2 class="govuk-heading-l"><%= t(".service_updates_title") %></h2>
+    <%= render ServiceUpdate::View.new(service_update: ServiceUpdate.first) %>
```
Comment (archferns) [inline app/views/pages/start.html.erb:12]:
```suggestion
    <%= render ServiceUpdate::View.new(service_update: ServiceUpdate.last) %>
```
Probably being a bit nitpicky here based on the logic, but `first` sounds counterintuitive to me, i.e not the last/latest

File: spec/components/service_update/view_preview.rb [hunk 2]
```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+module ErrorSummary
```
Comment (archferns) [inline spec/components/service_update/view_preview.rb:3]: Probably an incorrect namespace

**Review by archferns — APPROVED** [review]
Looking good, I've left a couple of nitpicky suggestions.

**Review by archferns — APPROVED** [review]

**Comment by mooktakim** [comment]
[[L] Build "service updates" homepage summary and section](https://trello.com/c/yMLVAa29/3120-l-build-service-updates-homepage-summary-and-section)

File: db/service_updates.yml [hunk 3]
```diff
@@ -0,0 +1,5 @@
+- id: 1
```
Comment (ralph-hawkins) [inline db/service_updates.yml:1]: @mooktakim can we the heading for the id so they are human readable please?

File: app/views/service_updates/index.html.erb [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+    href: root_path,
+  ) %>
+<% end %>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds-from-desktop">
+    <h1 class="govuk-heading-xl"><%= t(".heading") %></h1>
+
+    <h2 class="govuk-heading-m"><%= t(".contents") %></h2>
+    <ul class="govuk-list govuk-list--dash govuk-!-margin-bottom-8 govuk-!-margin-left-3">
```
Comment (ralph-hawkins) [inline app/views/service_updates/index.html.erb:15]: It doesn't look like the `govuk-list--dash` is rendering anything?

Screenshot 2021-11-11 at 10 41 49

Comment (ralph-hawkins) [reply app/views/service_updates/index.html.erb:15]: We might need to add our own class here

**Review by ralph-hawkins — CHANGES_REQUESTED** [review]
Can you add the styles:

`html {
  scroll-behavior: smooth;
}`

and

`p.govuk-body + ul.govuk-list--bullet {
  margin-top: govuk-spacing(-3);
}`


too please?

File: db/service_updates.yml [hunk 5]
```diff
@@ -0,0 +1,5 @@
...
+  date: '2021-09-01'
```
Comment (ralph-hawkins) [inline db/service_updates.yml:2]: Is it possible to set a default of the date this gets published?

File: db/service_updates.yml [hunk 6]
```diff
@@ -0,0 +1,5 @@
...
+  title: Lead and employing schools
```
Comment (ralph-hawkins) [inline db/service_updates.yml:3]: We've tweaked the content for this slightly:

title: You no longer need to provide a lead or employing school for trainees who are employed or privately funded

Content: >-
When you’re registering a trainee, we were asking for a lead or employing school in all cases. If a trainee is employed or funded privately we do not need this information so we’ve added a ‘not applicable’ option to the lead and employing schools sections. You do not need to provide lead or employing schools if the trainee is employed or funded privately.

File: db/service_updates.yml (hunk 5 above)
Comment (mooktakim) [reply db/service_updates.yml:2]: @ralph-hawkins 
As we are using YAML file, we won't be able to set the date automatically.
We'll have to set it manually each time.

File: db/service_updates.yml (hunk 3 above)
Comment (mooktakim) [reply db/service_updates.yml:1]: @ralph-hawkins
We can use text as ID also. I'll set it to "lead-employing-schools"

File: db/service_updates.yml (hunk 5 above)
Comment (ralph-hawkins) [reply db/service_updates.yml:2]: can we set this elsewhere and only overwrite it if there is a value in the YAML?

File: db/service_updates.yml (hunk 3 above)
Comment (ralph-hawkins) [reply db/service_updates.yml:1]: Can we set the ID to use the heading and apply a filter to convert it to kebab case please?

**Review by ralph-hawkins — APPROVED** [review]

**Pull request merged**